docs/content/latest/secure/authorization/rbac-model.md

```diff
@@ -44,6 +44,66 @@ Roles in YSQL can represent individual users or a group of users. They encapsula
 
 * Roles inherit the privileges of all other roles granted to them.
 
+View the roles for your YugabyteDB clusters with the following command:
+
+```sql
+yugabyte=> \du
+```
+
+```output
+                                     List of roles
+  Role name   |                         Attributes                         | Member of
+--------------+------------------------------------------------------------+-----------
+ postgres     | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+ yb_extension | Cannot login                                               | {}
+ yb_fdw       | Cannot login                                               | {}
+ yugabyte     | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+```
+
+The following table describes the default YSQL roles and users in YugabyteDB clusters.
+<!-- Portions of this table are also under Database authorization in Yugabyte cloud -->
+
+| Role | Description |
```

Comment:
So we have any more roles available in the cluster .  Use the command  select * from pg_roles;  to display all the roles. Is the intention to document all of those roles or  just which are added specifically by Yugabyte ?

Reply:
@sushantrmishra I think it's sufficient to document just what we add, plus the `postgres` user (to show it's got the same attributes as `yugabyte`), so \du seems more useful here. I'll add a suggestion a few lines up to clarify the wording, though.